datacats/cli/less.py

```diff
@@ -7,5 +16,17 @@ def less(environment, opts):
 ENVIRONMENT may be an environment name or a path to an environment directory.
 Default: '.'
 """
+    if not image_exists(LESSC_IMAGE):
+        sys.stdout.write('Pulling image {}'.format(LESSC_IMAGE))
+        for status in pull_stream(LESSC_IMAGE):
+            if 'status' not in status:
+                raise DatacatsError('Docker error: ' +
+                        (status['error'] if 'error' in status else 'Unknown error.'))
+            else:
+                sys.stdout.write('.')
+                sys.stdout.flush()
+        sys.stdout.write('\n')
+        sys.stdout.flush()
+
```

Comment:
don't duplicate this code, make a function that the pull command uses and call it instead